akka-docs/rst/java/code/docs/http/javadsl/server/HttpServerExampleDocTest.java

```diff
@@ -141,7 +141,9 @@ public TerminationDirective onUpstreamFailure(Throwable cause, Context<HttpReque
                 Flow.of(HttpRequest.class)
                     .via(failureDetection)
                     .map(request -> {
-                      Source<ByteString, Object> bytes = request.entity().getDataBytes();
+                      @SuppressWarnings("unchecked")
+                      Source<ByteString, Object> bytes =
+                          (Source<ByteString, Object>)(Source<ByteString, ?>) request.entity().getDataBytes();
                       HttpEntity.Chunked entity = HttpEntities.create(ContentTypes.TEXT_PLAIN_UTF8, bytes);
```

Comment:
aeh, this needs to be fixed: HttpEntity should accept `Source<ByteString, ? extends Object>`